wire-ios-data-model/Tests/Source/ManagedObjectContext/DatabaseMigrationTests.swift

```diff
@@ -428,7 +428,7 @@ final class DatabaseMigrationTests: DatabaseBaseTest {
 
             let store = NSManagedObjectModel(contentsOf: source.appendingPathComponent(modelFileName))!
             // then
-            XCTAssertTrue(store.versionIdentifiers.contains(version))
+            XCTAssertTrue(store.versionIdentifiers.contains(version), "\(version) should be contained")
```

Comment:
What should one do to correct this? I think stating that in the message would be helpful.

Reply:
added more details